ffi/dotnet/Devolutions.IronRdp/Devolutions.IronRdp.csproj

```diff
@@ -1,28 +1,45 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
-
   <PropertyGroup>
-    <TargetFramework>net8.0</TargetFramework>
-    <ImplicitUsings>enable</ImplicitUsings>
+    <PackageId>Devolutions.IronRdp</PackageId>
+    <Company>Devolutions</Company>
+    <Description>Bindings to Rust IronRDP native library</Description>
+    <LangVersion>latest</LangVersion>
+    <Version>2024.5.22.0</Version>
+    <ImplicitUsings>enable</ImplicitUsings> <!-- FIXME: set to disable -->
     <Nullable>enable</Nullable>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
     <SuppressDependenciesWhenPacking>false</SuppressDependenciesWhenPacking>
-
-    <PackageId>Devolutions.IronRdp</PackageId>
-    <Version>1.0.1</Version>
-    <Description>Dotnet bindings for the IronRDP project</Description>
   </PropertyGroup>
 
   <PropertyGroup>
     <RuntimesPath>../../../dependencies/runtimes</RuntimesPath>
     <NativeLibPath_win_x64>$(RuntimesPath)/win-x64/native/DevolutionsIronRdp.dll</NativeLibPath_win_x64>
+    <NativeLibPath_win_arm64>$(RuntimesPath)/win-arm64/native/DevolutionsIronRdp.dll</NativeLibPath_win_arm64>
+    <NativeLibPath_osx_x64>$(RuntimesPath)/osx-x64/native/libDevolutionsIronRdp.dylib</NativeLibPath_osx_x64>
+    <NativeLibPath_osx_arm64>$(RuntimesPath)/osx-arm64/native/libDevolutionsIronRdp.dylib</NativeLibPath_osx_arm64>
+    <NativeLibPath_osx_universal>$(RuntimesPath)/osx-universal/native/libDevolutionsIronRdp.dylib</NativeLibPath_osx_universal>
+    <NativeLibPath_linux_x64>$(RuntimesPath)/linux-x64/native/libDevolutionsIronRdp.so</NativeLibPath_linux_x64>
+    <NativeLibPath_linux_arm64>$(RuntimesPath)/linux-arm64/native/libDevolutionsIronRdp.so</NativeLibPath_linux_arm64>
+    <NativeLibPath_android_arm64>$(RuntimesPath)/android-arm64/native/libDevolutionsIronRdp.so</NativeLibPath_android_arm64>
+    <NativeLibPath_android_arm>$(RuntimesPath)/android-arm/native/libDevolutionsIronRdp.so</NativeLibPath_android_arm>
+    <NativeLibPath_android_x64>$(RuntimesPath)/android-x64/native/libDevolutionsIronRdp.so</NativeLibPath_android_x64>
+    <NativeLibPath_android_x86>$(RuntimesPath)/android-x86/native/libDevolutionsIronRdp.so</NativeLibPath_android_x86>
+    <NativeLibPath_ios_x64>$(RuntimesPath)/ios-x64/native/libDevolutionsIronRdp.dylib</NativeLibPath_ios_x64>
+    <NativeLibPath_ios_arm64>$(RuntimesPath)/ios-arm64/native/libDevolutionsIronRdp.dylib</NativeLibPath_ios_arm64>
+    <NativeLibPath_ios_universal>$(RuntimesPath)/ios-universal/native/libDevolutionsIronRdp.dylib</NativeLibPath_ios_universal>
+    <NativeLibPath_ios_framework>$(RuntimesPath)/ios-universal/native/libDevolutionsIronRdp.framework</NativeLibPath_ios_framework>
   </PropertyGroup>
 
+  <Import Condition="!$(PackageId.EndsWith('iOS'))" Project="Devolutions.IronRdp.Build.props "/>
+  <Import Project="Devolutions.IronRdp.Build.iOS.props "/>
+
+  <!--
```

Comment:
Seems like this is an artifact (it belongs, and exists in, the .targets file)

Reply:
I’ll remove it!